app/src/test/java/org/grammatek/simacorrect/spellcheckerservice/YfirlesturAnnotationTest.kt

```diff
@@ -0,0 +1,179 @@
+package org.grammatek.simacorrect.spellcheckerservice
+
+import com.google.common.truth.Truth.assertThat
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import org.grammatek.apis.DevelopersApi
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.io.File
+
+typealias AnnotationIndices = YfirlesturAnnotation.AnnotationIndices
+
+@Serializable
+data class TestCases (
+    val input: String,
+    val suggestions: List<String>,
+)
+
+@RunWith(RobolectricTestRunner::class)
+class YfirlesturAnnotationTest {
+    private val api = DevelopersApi("http://localhost:5002")
```

Comment:
Don't we want the CI to spin up its own Yfirlestur service for testing?

Reply:
Yes sure. But from the code alone it's not clear, how anybody on `locahost:5002` could possibly listen to the request. It would be helpful to describe the test scenario.

Reply:
Yeah that's fair, I'll have a small document for the tests in a couple of minutes for review